src/libs/ajax/leonardo/providers/LeoCookieProvider.ts

```diff
@@ -0,0 +1,26 @@
+import { LeoCookies } from 'src/libs/ajax/leonardo/LeoCookies';
+import { azureCookieReadyStore, cookieReadyStore } from 'src/libs/state';
+
+export interface LeoCookieProvider {
+  unsetCookies: () => Promise<void>;
+}
+
+export const leoCookieProvider: LeoCookieProvider = {
+  unsetCookies: async () => {
+    // TODO: call azure invalidate cookie once endpoint exists, https://broadworkbench.atlassian.net/browse/IA-3498
+
+    await LeoCookies()
+      .unsetCookie()
+      .catch((error) => {
+        // Suppress errors thrown by unsetCookie; if the call fails for auth or CORS reasons signout logic should proceed
+        if (error instanceof Response && error.status === 401) {
+          // Ignore: user already logged out
+        } else {
+          console.error('Unexpected error unsetting user cookie', error);
```

Comment:
This is the change which appears to prevent PROD-963 issue (users can't log in): the call to invalidateToken returns a 401, the browser complains because the response violates CORS, throwing a TypeError instead of a Response: 401, and this conditional block would rethrow the error, preventing signout from completing. Suppress all errors here instead, and the cookie is unset and the user is shown the readable "Sessions last 24 hours" toast.